### ISSUE TITLE
Search Page - Request button persists when there are 0 matches

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#2213]
On the search page /talent/search when there are 0 candidates that match the filters the "We can still help!" section shows up properly, but I think the "Request Candidates" section for the specific pool is supposed to be hidden (you can't request 0 candidates).

This was tested on UAT server.

@Jerryescandon please confirm.

**Comment by Jerryescandon** [comment]
Agreed, when there are no candidates, the "Request Candidates" section should be hidden.
